src/status_im/contexts/wallet/add_account/add_address/confirm_address/view.cljs

```diff
@@ -0,0 +1,110 @@
+                                                                              {:address    address
+                                                                               :public-key ""}])
+                                                         :save  (rf/dispatch
+                                                                 [:wallet/save-address
+                                                                  {:address address
+                                                                   :name @account-name
+                                                                   :customization-color @account-color
+                                                                   :ens (when ens? address)
+                                                                   :on-success
+                                                                   (fn []
+                                                                     (rf/dispatch [:navigate-back])
+                                                                     (debounce/debounce-and-dispatch
+                                                                      [:navigate-back]
+                                                                      sheet-closing-delay)
+                                                                     (debounce/debounce-and-dispatch
+                                                                      [:toasts/upsert
+                                                                       {:type  :positive
+                                                                        :theme theme
+                                                                        :text  (i18n/label
+                                                                                :t/address-saved)}]
+                                                                      sheet-closing-delay))}])))}}
+         [quo/data-item
+          {:card?           true
+           :right-icon      :i/advanced
+           :icon-right?     true
+           :emoji           @account-emoji
+           :title           (-> confirming-addresses-purposes
+                                (get-in [purpose :address-type])
+                                i18n/label)
+           :subtitle        address
+           :status          :default
+           :size            :default
+           :subtitle-type   :default
+           :custom-subtitle (fn [] [quo/text
+                                    {:size   :paragraph-2
+                                     ;; TODO: monospace font
+                                     ;; https://github.com/status-im/status-mobile/issues/17009
+                                     :weight :monospace}
+                                    address])
+           :container-style style/data-item
+           :on-press        #(js/alert "To be implemented")}]]]])))
```

Comment:
you could define it outside, to show the right pattern, even if it's not implemented :)